bosh-delete-all-deployments/task

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -exu
+
+# shellcheck disable=SC1091
+source cf-deployment-concourse-tasks/shared-functions
+
+function bosh_delete_all_deployments() {
+  local force_flag=""
+
+  if [ "$IGNORE_ERRORS" = true ]; then
+    force_flag="--force"
+  fi
+
+  local deployments
+  deployments=$(bosh deployments --json | jq -r '.Tables[].Rows[].name')
+
+  for deployment in ${deployments}; do
+    if [ ! -z "${deployment}" ]; then
```

Comment:
Use `-n` instead of `! -z`.

Reply:
@davewalter Done!

Also - when did you move to RelInt?!?! So exciting!